[2025-02-28T02:17:34.079+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-28T02:17:34.105+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: json_ingestion_s3_partitioned.upload_events scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-02-28T02:17:34.118+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: json_ingestion_s3_partitioned.upload_events scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-02-28T02:17:34.119+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-28T02:17:34.140+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_events> on 2025-02-27 00:00:00+00:00
[2025-02-28T02:17:34.147+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7433) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-28T02:17:34.149+0000] {standard_task_runner.py:72} INFO - Started process 7434 to run task
[2025-02-28T02:17:34.147+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'json_ingestion_s3_partitioned', 'upload_events', 'scheduled__2025-02-27T00:00:00+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpsjo3qs9i']
[2025-02-28T02:17:34.150+0000] {standard_task_runner.py:105} INFO - Job 65: Subtask upload_events
[2025-02-28T02:17:34.163+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-28T02:17:34.199+0000] {task_command.py:467} INFO - Running <TaskInstance: json_ingestion_s3_partitioned.upload_events scheduled__2025-02-27T00:00:00+00:00 [running]> on host be61fafefd76
[2025-02-28T02:17:34.201+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-28T02:17:34.416+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='json_ingestion_s3_partitioned' AIRFLOW_CTX_TASK_ID='upload_events' AIRFLOW_CTX_EXECUTION_DATE='2025-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-27T00:00:00+00:00'
[2025-02-28T02:17:34.417+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-28T02:17:34.440+0000] {logging_mixin.py:190} INFO - Partition Path: 2025-02-27/
[2025-02-28T02:17:34.442+0000] {logging_mixin.py:190} INFO - Uploading events.json to users_events/2025-02-27/events.json
[2025-02-28T02:17:45.549+0000] {logging_mixin.py:190} INFO - Error uploading events.json: Could not connect to the endpoint URL: "https://dataingestionjson.s3.amazonaws.com/users_events/2025-02-27/events.json"
[2025-02-28T02:17:45.551+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-28T02:17:45.619+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-28T02:17:45.623+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=json_ingestion_s3_partitioned, task_id=upload_events, run_id=scheduled__2025-02-27T00:00:00+00:00, execution_date=20250227T000000, start_date=20250228T021734, end_date=20250228T021745
[2025-02-28T02:17:45.838+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-28T02:17:46.021+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-28T02:17:46.035+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
